internal/node/services.go

```diff
@@ -99,7 +102,23 @@ func newSupervisorService(
 		s = append(s, newAuthorityClaimer(c, workDir))
 	}
 
-	s = append(s, newHttpService(c))
+	// initialize machines for inspect
+	repo := &repository.MachineRepository{Database: database}
+
+	machines, err := machines.Load(context.Background(), repo, c.MachineServerVerbosity)
+	if err != nil {
+		slog.Error("failed to load the machines", "error", err)
+		os.Exit(1)
+	}
+	defer machines.Close()
+
+	inspect, err := inspect.New(machines)
+	if err != nil {
+		slog.Error("failed to create the inspect", "error", err)
+		os.Exit(1)
+	}
```

Comment:
```suggestion
	// initialize machines for inspect
	repo := &repository.MachineRepository{Database: database}

	machines, err := machines.Load(context.Background(), repo, c.MachineServerVerbosity)
	if err != nil {
		slog.Error("failed to load the machines", "error", err)
		os.Exit(1)
	}
	defer machines.Close()

	inspector, err := inspect.New(machines)
	if err != nil {
		slog.Error("failed to create inspector", "error", err)
		os.Exit(1)
	}
```

I still have to see the advancer initialization code but, perhaps we could move part of this to a separate function responsible for creating the service